components/src/core/components/CardTable/Cell/Checkbox.vue

```diff
@@ -75,4 +104,4 @@ export default defineComponent({
 });
 </script>
 
-<style src="./default.scss" lang="scss" scoped></style>
+<style src="./default.scss" lang="scss"></style>
```

Comment:
any reasons to remove scoped?

Reply:
yeah, when scoped the styles I added are not affected to inner oxd-checkbox. since this stylesheet only has specific styles, i hope it won't be an issue.